blackbox/docs/general/blobs.rst

```diff
@@ -54,11 +54,11 @@ custom path globally for all blob tables::
 
 Also see :ref:`config`.
 
-Per Blob Table Setting
+Per blob table setting
 ----------------------
 
 It is also possible to define a custom blob data path per table instead of
-global by config. Also per table setting take precedence over the config
+global by config. Also per table settings take precedence over the config
```

Comment:
this should be reverted because it's not related to the PR

Reply:
done